### PR TITLE
Bug/#208 static java class misses properties

### LINKED ISSUE #208: [BUG] Static java class for extended entity does not show all properties
**Describe the bug**
If you create an Aspect model with an Entity `A` extending Entity `B` and generate the static Java classes for it, the meta class for `A` is missing a method to get all properties for `A` and `B`. There is currently only a method `getProperties()` which is returning the properties of `A`.

### PR DESCRIPTION
**Pull request by DvSmi**
fixes #208

### CODE REVIEW

File: core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+    # Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
```
Comment (atextor) [inline core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl:1]: Remove unnecessary indentation. Also see other model files below, this occurs multiple times.

File: core/sds-aspect-model-java-generator/src/main/resources/java-static-class-body-lib.vm [hunk 2]
```diff
@@ -75,6 +75,19 @@ static {
        );
     }
 
+    @Override
+    public List<StaticProperty<?>> getAllProperties() {
+        #if( $element.getExtends().isPresent() )
+            #set( $extendedElement = $element.getExtends().get() )
+            $codeGenerationConfig.getImportTracker().importExplicit( "${codeGenerationConfig.getPackageName()}.Meta${extendedElement.getName()}" )
+            List<StaticProperty<?>> properties = getProperties();
+            properties.addAll(Meta${extendedElement.getName()}.INSTANCE.getProperties());
```
Comment (atextor) [inline core/sds-aspect-model-java-generator/src/main/resources/java-static-class-body-lib.vm:84]: I think this method does not handle multiple steps of inheritance, i.e. grand-parent's (and higher) properties. Shouldn't this line have `INSTANCE.getAllProperties()`?

Additionally, I think there is no test for this case.

**Review by atextor — CHANGES_REQUESTED** [review]

File: core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl (hunk 1 above)
Comment (DvSmi) [reply core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl]: done

File: core/sds-aspect-model-java-generator/src/main/resources/java-static-class-body-lib.vm (hunk 2 above)
Comment (DvSmi) [reply core/sds-aspect-model-java-generator/src/main/resources/java-static-class-body-lib.vm]: Corrected.

There is a test where we validate against the method of the generated files, including one for a grand-parent.

**Review by atextor — APPROVED** [review]

**Pull request merged**